packages/core-types/src/event.rs

```diff
@@ -1,203 +0,0 @@
-use std::{any::Any, cell::RefCell, rc::Rc};
```

Comment:
This showed up in search results when I was trying to edit the event type, but this version is entirely unused